lib/experimental-default-theme.json

```diff
@@ -131,6 +131,42 @@
 						"slug": "huge",
 						"value": 48
 					}
+				],
+				"fontFamilies": [
```

Comment:
By reading [this](https://github.com/WordPress/gutenberg/issues/25032#issuecomment-686318340) my understanding is that we want to ship only the `System Font`?

Reply:
I added the fonts on this list provided by @jasmussen https://github.com/WordPress/gutenberg/pull/24868#issuecomment-685377191.
@mtias can we use this small list of websafe safe fonts or you have strong preference for keeping only a websafe font?
I would prefer to have at least two or three so by default users have a choice.

Reply:
Indeed, those are two separate directions.

How about we keep this PR as small as can be, and go with the Matías direction for now, and then move the addition of font families to a separate followup PR? 

One aspect of Matías comment which I overlooked in providing the intitial list is that perhaps it really is best if the _theme_ provides the list(s) of available fonts, rather than WordPress itself.

Reply:
Yes, I'd prefer to keep this systems only for now until we figure out how themes would interact with it.